bin/picca_Pk1D_postprocess.py

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/env python
+"""Compute the averaged 1D power spectrum
+"""
+
+import sys, os, argparse
+import numpy as np
+import fitsio
+from picca.pk1d import postproc_pk1d, postproc_pk1d_oldoutput
+
+
+def define_wavenumber_array(k_min, k_max, k_dist, velunits, pixsize, rebinfac):
+    """ Define the wavenumber array limits and binning
+        Default binning defined linearly with assumed average redshift of 3.4,
+        and a forest defined between 1050 and 1200 Angstrom.
+        Default velocity binning with same number of pixels"""
+
+    k_min_lin_default = 2 * np.pi / ((1200 - 1050) * (1 + 3.4) / rebinfac)
```

Comment:
Hardcoded numbers were set in `picca.pk1d.utils`
At this point I don't change the choice of default parameters (@corentinravoux told me it's essentially from you, and I agree it's a good starting point at least in AA). We certainly need to discuss this with you and Naim.
This can be done later, for another PR.